algorithms/adaboost.py

```diff
@@ -1,7 +1,12 @@
 import numpy as np
+from sklearn import datasets

```

Comment:
please move all testing relevant imports into the if __name__ == "__main__". same in all other files